src/slic3r/GUI/ConfigWizard.cpp

```diff
@@ -65,12 +65,12 @@ using Config::SnapshotDB;
 
 
 // Configuration data structures extensions needed for the wizard
-//BBS: set BBL as default
-bool Bundle::load(fs::path source_path, bool ais_in_resources, bool ais_bbl_bundle)
+// SM_FEATURE: set sm machine as default
```

Comment:
keep bbl either

Reply:
The two function has the same implement. The function depends the default machine you choose in code. So, the "Set BBL as default" version is meaningless in this context. In fact, "ais_default_bundle" is the more appropriate than the name "ais_bbl_bundle".